bulk.yml

```diff
@@ -2,10 +2,12 @@
 continue: "true"
 transfer: "ln_s"
 exclude: "clientpath"
-checksum_algorithm: "File-Size-64"
-skip: "minmax"
+skip: "all"
```

Comment:
This will require a build with https://github.com/openmicroscopy/openmicroscopy/pull/5850 to work as expected